os/tools/mkheader.py

```diff
@@ -23,10 +23,10 @@
 #
 # header information :
 #
-# / header size (2byte) / binary type (2byte) / binary size (4byte)
-# / Binary name (16byte) / Binary version (16byte)
-# / Kernel version (8byte) / Jump address (4byte)
-# header size is 52byte.
+# / Header size (2bytes) / Binary type (2bytes) / Binary size (4bytes)
```

Comment:
Could you use Jeongchan's way to show packet information as shown below
```
	 * +------------------------------------------+
	 * | sender_pid | msg type |     message      |
	 * | (4bytes)   | (4bytes) | (Max 65527bytes) |
	 * +------------------------------------------+
```

Reply:
I modified.